css/carliebydani.css

```diff
@@ -8,55 +8,88 @@
 
 @layer utilities {
     .wrapper {
-        @apply container
-            max-w-7xl
-            mx-auto
-            p-2;
+        @apply container max-w-7xl mx-auto p-2;
     }
 
     .body {
-        @apply bg-[#262626]
-        text-white;
+        font-family: 'Sofia Sans', sans-serif;
+        @apply bg-[#262626] text-white;
     }
 
     .header {
         @apply bg-black;
-    }
 
-    .top {
-        @apply w-full
-        border-b
-        border-b-red-500;
-    }
+        .top {
+            @apply w-full border-b border-b-red-500;
 
-    .top-container {
-        @apply wrapper
-        flex
-        flex-col
-        items-center
-        md:flex-row
-        md:justify-between;
-    }
+            .top-container {
+                @apply wrapper flex flex-col items-center md:flex-row md:justify-between;
 
-    .top-tags {
-        @apply flex
-        items-center;
-    }
+                li:hover {
+                    @apply text-red-500 cursor-pointer transition duration-300 ease-in-out;
+                }
 
-    .main-nav {
-        @apply wrapper
-        flex
-        justify-between;
-    }
+                .top-tags {
+                    @apply flex items-center;
+                }
+
+                button {
+                    @apply py-2 px-4 flex gap-2 bg-[#505050] text-sm font-bold uppercase;
+
+                    :first-child {
+                        @apply w-5 h-5 block bg-[url('http://carlifebydani.local/wp-content/uploads/2023/12/red-heart.png')] bg-contain bg-no-repeat;
+                    }
+
+                    &:hover {
+                        @apply bg-red-500;
+
+                        :first-child {
+                            @apply bg-[url('http://carlifebydani.local/wp-content/uploads/2023/12/white-heart.png')];
+                        }
+                    }
+                }
+            }
+        }
+
+        .main-nav {
+            @apply wrapper flex justify-between;
+
+            ul {
+                @apply flex items-center;
+
+                li {
+                    @apply text-xl font-bold w-max;
+
+                    &:hover {
+                        @apply text-red-500 cursor-pointer transition duration-300 ease-in-out;
+                    }
+                }
+            }
+
+            .sidebar {
+                @apply flex flex-col items-start gap-1;
+
+                .sidebar-item {
+                    @apply flex flex-col gap-1;
+
+                    .item-title {
+                        @apply text-[17px] font-bold;
```

Comment:
avoid using sizes in pixels
We have approval from the designer that we can refer to the sm, lg... etc classes if it looks well